substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -485,6 +485,12 @@ pub mod pallet {
 	#[pallet::getter(fn force_era)]
 	pub type ForceEra<T> = StorageValue<_, Forcing, ValueQuery>;
 
+	/// Maximum staked rewards, i.e. the percentage of the era inflation that
+	/// is used for stake rewards.
+	/// See [Era payout](./index.html#era-payout).
+	#[pallet::storage]
+	pub type MaxStakedRewards<T> = StorageValue<_, Percent, ValueQuery>;
```

Comment:
One of the goals of this change is to expose is to expose the minimum % of the era inflation that goes directly into the treasury, thus the storage item instead of the runtime parameters.